src/Constants.cpp

```diff
@@ -0,0 +1,20 @@
+#include "Constants.h"
+
+#include "Globals.h"
+
+namespace Constants {
+constexpr std::array<int32_t, 4> NORMAL_WHEEL_ROTS = {0, 0, 0, 0};
+const swervewheelvel_t TURN_IN_PLACE_VEL =
+	kinematics::SwerveDriveKinematics(Constants::WHEEL_BASE, Constants::ROBOT_LENGTH)
```

Comment:
We should probably reuse `robot::swerveKinematics()` instead of re-constructing the swerve kinematics